app/components/benchmark_banner_component/benchmark_banner_component.scss

```diff
@@ -0,0 +1,58 @@
+
+.benchmark-banner-wrapper {
+
+  .benchmark-banner {
+    background-color: $ncce-highlight;
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+
+    .benchmark-banner__label {
+      flex-grow: 0;
+      flex-shrink: 1;
+      display: flex;
+      align-items: center;
+      padding-left: 10px; 
+    }
+
+    .benchmark-banner__icons {
+      flex-grow: 1;
+      display: flex;
+      flex-direction: row;
+      gap: 10px;
+      padding: 0 10px;
+
+      &--align-right {
+        justify-content: flex-end;
+      }
+    }
+
+  }
+}
+
+.benchmark-banner--width-float {
+  width: 100%;
+
+  .benchmark-banner {
+    margin-top: 10px;
+  }
+
+  @include govuk-media-query($from: desktop) {
+    width: calc(((100vw - 960px)/2) + (960px * 0.333) - 30px);
```

Comment:
There are a lot of magical constants here. It would be really nice if we could have the width determined by the parent element.

If not, it would be great if this maths could be simplified. I believe the following is what is evaluates to:
```
width: calc(50vw - 190px);
```